src/PDO/Peachpie.Library.PDO/PDOStatement.cs

```diff
@@ -32,25 +32,37 @@ public class PDOStatement : IPDOStatement, IDisposable
         private bool m_namedAttr = false;
         private Dictionary<string, string> m_namedPlaceholders;
         private List<String> m_positionalPlaceholders;
+
+        private bool m_hasParamsBounded = false;
```

Comment:
can we just remove this field and replace it with a property like
`bool HasParamsBound => m_boundedParams != null;`